tools/e2etesting/TestEventProcessor/TestEventProcessor.Businesslogic/StopResult.cs

```diff
@@ -13,6 +16,31 @@ public class StopResult : IResult
         /// <summary>
         /// Flag whether the monitoring was successful (without errors) or not.
         /// </summary>
-        public bool IsSuccess { get; set; }
+        public bool? UnexpectedTimestamps { get; set; }
```

Comment:
>? [](start = 19, length = 1)

why using nullable bool 
true --> unexpected timestamps
false --> only expected timestamps
null --> ?

Reply:
Removed the property completely for now.